### PR TITLE
feat(TMRX-1400): Implement category labels flags above the headline

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
### Description
Implement labels to the top of the tiles (above the headline) per the updated design

Expirable flag , article.label or article content type

Technical details 
We need to enable all the tag and flag types on ALL article tiles in the slices. (Link to design with all potential labels https://www.figma.com/file/zYjnqWXVY0cFFaJgcSqM5M/Section-Slices?type=design&node-id=3065%3A242040&mode=design&t=SDQbHy7W2FD2Bdk4-1 - Connect your Figma account  )

Acceptance Criteria

Implement labels to the top of the tiles (above the headline) per the updated design

[JIRA-1400](https://nidigitalsolutions.jira.com/browse/TMRX-1400)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):

Screenshot 2023-08-16 at 16 08 21

### CODE REVIEW

File: packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+export const ArticleTileInfo = ({
+  expirableFlag,
+  contentType,
+  articleLabel,
+  marginBlockStart = 'space000',
+  marginBlockEnd = 'space000'
+}: ArticleTileInfoProps) => {
+  const hasTag = Boolean(contentType);
+  const hasExpirableFlag = Boolean(expirableFlag && expirableFlag !== '');
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:62]: Just for own understanding: what `hasExpirableFlag` is for?

File: packages/ts-newskit/src/assets/VideoIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+const VideoIcon = (props: any) => (
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/assets/VideoIcon.tsx:3]: I know we have lots of `any` in our codebase but I think this one should be `SvgProps`

File: packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx (hunk 1 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:62]: i just named those props based on design here is the design file https://www.figma.com/file/zYjnqWXVY0cFFaJgcSqM5M/Section-Slices?node-id=3065%3A242040&mode=dev
Screenshot 2023-08-17 at 10 12 54

Comment (yukiinasawa) [reply packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:62]: Ah okay cool!

**Review by yukiinasawa — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/shared-styles/index.ts [hunk 3]
```diff
@@ -70,3 +72,21 @@ export const StyledSpan = styled.span<{ hasCaption: boolean }>`
...
+export const StyledBlock = styled(Block)<{ hasVideoIcon?: boolean }>`
+  display: ${({ hasVideoIcon }) => (hasVideoIcon ? 'flex' : 'inline-block')};
+  align-items: ${({ hasVideoIcon }) =>
+    hasVideoIcon ? 'last baseline' : 'initial'};
+`;
+
+export const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 4px;
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/shared-styles/index.ts:91]: token vale? `space010`

File: packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  contentType?: string;
+  articleLabel?: string;
+  marginBlockEnd?: string;
+  marginBlockStart?: string;
+}
+
+const CustomTextBlock = ({
+  text,
+  stylePreset,
+  hasVideoIcon
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:18]: maybe we should extend to allow any icon to be used and in this case pass the icon in?

File: packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            text={expirableFlag}
+          />
+        )
+      )}
+      {contentType && expirableFlag && <CustomDivider />}
+      {contentType && (
+        <CustomTextBlock text={contentType} hasVideoIcon={hasVideoIcon} />
+      )}
+      {articleLabel && <CustomDivider />}
+      {articleLabel && <CustomTextBlock text={articleLabel} />}
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:94]: could do (may need to wrap with Fragment);
```
{articleLabel && (
     <CustomDivider />
     <CustomTextBlock text={articleLabel} />
)}
```

**Review by douglasmik — COMMENTED** [review]
Could we add options for these fields in the slices?

File: packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx (hunk 4 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:18]: make sense

File: packages/ts-newskit/src/components/slices/shared-styles/index.ts (hunk 3 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/shared-styles/index.ts:91]: done

File: packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx (hunk 4 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/shared/articleTileInfo.tsx:18]: done

**Review by ibrahim-kurhan — COMMENTED** [review]
@douglasmik I added them as options as well in slices
Screenshot 2023-08-17 at 15 08 25

**Pull request merged**